services/cards-publication/src/main/java/org/opfab/cards/publication/services/CardValidationService.java

```diff
@@ -117,14 +117,25 @@ void validate(Card c) throws ConstraintViolationException {
                     "constraint violation : forbidden characters ('#','?','/') in process or processInstanceId", null);
     }
 
+    void validateCardForPatch(Card cardForPatch, Card initialCard) throws ConstraintViolationException {
+        if ((cardForPatch.getProcess() != null) && !cardForPatch.getProcess().equals(initialCard.getProcess()))
+            throw new ConstraintViolationException("The current process field " + initialCard.getProcess()
+                    + " can not be patched with new value " + cardForPatch.getProcess(), null);
```

Comment:
Proposal : 

```
String.format("The current process field %s cannot be patched with new value %s",initialCard.getProcess(),  cardForPatch.getProcess())
``` 
```

Reply:
After discussion, we've decided to keep it as it is.